src/main/resources/application.properties

```diff
@@ -19,4 +19,7 @@ spring.jpa.properties.hibernate.connection.driver_class=org.postgresql.Driver
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.show-sql = true
 spring.jpa.hibernate.ddl-auto = update
-spring.jpa.properties.hibernate.default_schema=public
\ No newline at end of file
+spring.jpa.properties.hibernate.default_schema=public
+spring.servlet.multipart.enabled=true
+spring.servlet.multipart.max-file-size=300MB
+spring.servlet.multipart.max-request-size=300MB
```

Comment:
Is this too large? Not sure what the standard is.

Reply:
Yes this is too large. I would start at 10MB. A normal image size is like <100KB